jhub_apps/templates/style.css

```diff
@@ -22,12 +22,13 @@
     --link-text-color: {{ text_color | default("#1c1d26") }};
     --primary-color: {{ primary_color | default("#ba18da") }};
     --primary-color-dark: {{ primary_color_dark | default("#9b00ce") }};
+    --primary-color-light: {{ primary_color_light | default("#ba18da10") }};
```

Comment:
I see you added this, but its not being used anywhere. Also, if planning to use this, Nebari does not currently provide primary_color_light as a variable, so if you need this we should discuss adding it to Nebari as well.